src/sections/Community/Handbook/about.js

```diff
@@ -8,22 +8,72 @@ import Bullet from "./images/active.png";
 import TOC from "../../../components/handbook-navigation/index";
 
 
+const IntroWrapper = styled.div`
+    padding: 3rem 20rem;
+    margin-top: -25rem; 
+    h1{
+      margin-bottom: 2rem;
+    }
+    h2{
+      color:black;
```

Comment:
Why these colors are being added again?

Reply:
@Nikhil-Ladha i think what he's working on has already been fixed, i think. And also the wrapper he's working on has been deleted already